packages/common/src/services/integrations/jwp/JWPAccountService.ts

```diff
@@ -61,7 +61,7 @@ export default class JWPAccountService extends AccountService {
       canUpdateEmail: false,
       canSupportEmptyFullName: false,
       canChangePasswordWithOldPassword: true,
-      canRenewSubscription: false,
+      canRenewSubscription: true,
```

Comment:
It looks like it can't be done for JWP now.

Reply:
Yes, this should be reverted and solved with a `canSwitchSubscription` flag instead.